userspace/engine/falco_utils.h

```diff
@@ -26,6 +26,17 @@ limitations under the License.
 
 #pragma once
 
+#if (__cplusplus < 201402L)
```

Comment:
We should already have `absl::make_unique` available if I don't remember it wrong.

This way we could avoid the polyfill here.

Would you please look into it?

Reply:
I can't find it used anywhere. I do see it as a gRPC submodule, meaning its use would conflict MINIMAL_BUILD.
Do you prefer to use abseil and have it as a separate ExternalProject?

Reply:
Right! I was not remembering that it comes as gRPC submodule...

In this case, I'd left your polyfill.

Anyways, if we're talking about what I prefer. Well, I'd prefer to port the whole codebase to C++14 (or greater). 🤣